src/Pages/Nodes.razor

```diff
@@ -179,7 +179,7 @@
 
     private async Task GetData()
     {
-        _nodes = await NodeRepository.GetAll();
+        _nodes = await NodeRepository.GetAllManagedByFundsManager();
```

Comment:
Not managed nodes should not be visible in the Nodes view, but they should be selectable from the channel request view